### PR TITLE
add augmentation ops

### PR DESCRIPTION
**Pull request by wangxinxin08**
fix bugs

### CODE REVIEW

File: ppdet/data/transform/operators.py [hunk 1]
```diff
@@ -2555,3 +2555,421 @@ def __call__(self, sample, context=None):
         save_path = os.path.join(self.output_dir, out_file_name)
         image.save(save_path, quality=95)
         return sample
+
+
+@register_op
+class Rotate(BaseOperator):
```
Comment (heavengate) [inline ppdet/data/transform/operators.py:2561]: 注释加一下说明和参数定义之类的吧

Comment (wangxinxin08) [reply ppdet/data/transform/operators.py:2561]: 好的

**Review by qingqing01 — COMMENTED** [review]
LGTM.

And can add a doc for visualized effect for different data aug in the future

**Review by heavengate — APPROVED** [review]
God job

**Pull request closed without merging**